consumer.go

```diff
@@ -0,0 +1,286 @@
+package memphis
```

Comment:
at the JS there is ping consumer logic that not exist in here, so please add it
and docs as well